Library/Homebrew/cmd/test-bot.rb

```diff
@@ -98,6 +103,10 @@ def puts_command
     end
 
     def puts_result
+      if ENV["TRAVIS"]
+        puts "travis_fold:end:#{@command.join(".")}"
+        return
```

Comment:
The `return` here is unnecessary and it will prevent the status result being printed.